packages/jest/.eslintrc.json

```diff
@@ -13,7 +13,7 @@
       "files": ["**/*.ts"],
       "excludedFiles": ["./src/migrations/**"],
       "rules": {
-        "no-restricted-imports": ["error", "@nrwl/workspace"]
+        "no-restricted-imports": ["error"]
```

Comment:
@FrozenPandaz - had to take off this `no-restricted-imports` to import the `readWorkspaceConfig()` function. Let me know if I'm missing something (not sure why workspace is restricted?) - and also disabling the next line for the specific import statements is preferred!

Reply:
You can't import from `@nrwl/workspace` because we have some utilities that use `@angular-devkit` which isn't installed for some workspaces. We need this check.

Let's do a deep import for now.